lib/database/migrations/20231025101611-create-lhc-periods-view.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+const CREATE_PERIODS_VIEW = `
+CREATE OR REPLACE VIEW lhc_periods_statistics AS
+SELECT  
+    p.id,
+    AVG(r.lhc_beam_energy) AS avgEnergy
+FROM lhc_periods AS p
+LEFT JOIN (
+    SELECT * FROM runs
+        WHERE
+            definition = 'PHYSICS'
+    ) AS r
```

Comment:
No reason to use a subquery here, you should directly put the condition in the join.
Subqueries are useful when you have a limit/an order in it.

I did not tested the case here, but in general subqueries are quite expensives in mariadb (you can check with an explain)

Reply:
here, its good point, refactored